src/components/DisplayAlgoLine.js

```diff
@@ -0,0 +1,4 @@
+const DisplayAlgoLine = ({ height }) => {
+  return <div className='DisplayAlgoLine' style={{ height: height }}></div>;
+};
```

Comment:
This simple component is a good room to showcase two syntax improvements you could apply to other places:

1) Shortened [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#comparing_traditional_functions_to_arrow_functions):

When an arrow function's body contains an immediate return of a value, you can omit the braces and the return keyword, as shown below:

```suggestion
const DisplayAlgoLine = ({ height }) => <div className='DisplayAlgoLine' style={{ height: height }}></div>;
```

2) When defining an object, if a given property should have the same name of a variable you are assigning to it, you can shortcut it's assigning as follows:

```suggestion
const DisplayAlgoLine = ({ height }) => {
  return <div className='DisplayAlgoLine' style={{ height }}></div>;
};
```